nxviz/plots.py

```diff
@@ -219,14 +224,15 @@ def draw(self):
             self.draw_group_labels()
         logging.debug("DRAW: {0}".format(self.sm))
         if self.sm:
-            self.figure.subplots_adjust(right=0.8)
-            cax = self.figure.add_axes([0.85, 0.2, 0.05, 0.6])
-            self.figure.colorbar(self.sm, cax=cax)
+            #self.figure.subplots_adjust(right=0.8)
+            #cax = self.figure.add_axes([0.85, 0.2, 0.05, 0.6])
+            #self.figure.colorbar(self.sm, cax=cax)
```

Comment:
After uncommenting the above three lines, have you run tests locally on your computer to check that the tests pass?

Reply:
The three commented lines are the original ones (Sorry, I commented instead of deleting them). If, as I suggest they are replaced by:
```suggestion
            self.figure.colorbar(self.sm, ax=self.ax)
```
thus plotting to ax and not to a newly created cax it works also for several subplots in one figure. But the colorbars look a bit narrower. I will add some examples to the test suite.

Reply:
Got it! Yes, please add a few tests.

Btw, I would avoid putting actual plotting code in the test suite. I have not yet set up plotting tests properly. You can test for logical properties of the objects instead.